telegram-bot/src/main/java/com/exadel/telegrambot/bot/utils/Urls.java

```diff
@@ -0,0 +1,15 @@
+package com.exadel.telegrambot.bot.utils;
+
+public interface Urls {
+
+    String TOKEN="bot"+ Security.TOKEN +"/";
+    String TELEGRAM_BASE="https://api.telegram.org/";
+
+    String BASE_WEBHOOK="api/telegram";
+    String FULL_REQUEST=TELEGRAM_BASE+TOKEN;
+
+
```

Comment:
Please remove extra empty space one line is enough

Reply:
done